src/firefly/js/ui/ImageSelect.jsx

```diff
@@ -115,12 +115,19 @@ function FilterPanel({imageMasterData, title='Select Data Set', onChange}) {
     );
 }
 
+function filter(t){
```

Comment:
is this function used? If not, can you remove it before merging?

Reply:
No, I will take it out now.